src/Components/reusables/DiveLogImageCarousel.tsx

```diff
@@ -101,76 +99,27 @@ const DiveLogImageCarousel: FunctionComponent<
   };
 
   const [focusedImageIndex, setFocusedImageIndex] = React.useState(0);
-  if (props.images && props.images.length) {
-    return (
-      <View style={styles.header}>
-        <ScrollView
-          horizontal
-          pagingEnabled
-          onScroll={handleScroll}
-          showsHorizontalScrollIndicator={false}>
-          {props.images.map((image, index) => (
-            <Image
-              key={index}
-              style={styles.headerImage}
-              source={{ uri: image.uri }}
-            />
-          ))}
-        </ScrollView>
-        <View style={styles.headerIconsContainer}>
-          <TouchableWithoutFeedback onPress={props.goBack}>
-            <View style={styles.headerIcon}>
-              <FEIcon name="chevron-left" color="black" size={25} />
-            </View>
-          </TouchableWithoutFeedback>
-          {props.shareUrl ? (
-            <TouchableWithoutFeedback onPress={onShare}>
-              <View style={styles.headerIcon}>
-                <FEIcon name="share" color="black" size={25} />
-              </View>
-            </TouchableWithoutFeedback>
-          ) : (
-            <TouchableWithoutFeedback>
-              <View style={styles.headerIcon}>
-                <FEIcon name="share" color="black" size={25} />
-              </View>
-            </TouchableWithoutFeedback>
-          )}
-        </View>
-        <View style={styles.headerBottomContainer}>
-          <View style={styles.photoDots}>
-            {props.images.map((_dot, index) => {
-              return (
-                <View
-                  key={index}
-                  style={
-                    index === focusedImageIndex
-                      ? styles.whitePhotoDot
-                      : styles.blackPhotoDot
-                  }
-                />
-              );
-            })}
-          </View>
-          <View style={styles.imageCountContainer}>
-            <Icon name="image-outline" size={18} color="#FFF" />
-            <Text style={styles.imageCountText}>{`${focusedImageIndex + 1}/${
-              props.images.length
-            }`}</Text>
-          </View>
-        </View>
-      </View>
-    );
-  }
+  const images = (props.images && props.images.length)
+    ? props.images
+    : defaultImages
   return (
     <View style={styles.header}>
       <ScrollView
         horizontal
         pagingEnabled
         onScroll={handleScroll}
         showsHorizontalScrollIndicator={false}>
-        {defaultImages.map((image, index) => (
-          <Image key={index} style={styles.headerImage} source={image.source} />
+        {images.map((image, index) => (
+          <Image
+            key={index}
+            style={[
+              styles.headerImage,
+              !props.images ? {
+                height: 130
```

Comment:
Can you test this piece of code on logs where there are only default images and let me know what you find?

Reply:
ah you're right, good catch